modules/config.js

```diff
@@ -125,6 +126,25 @@ module.exports = {
     
         let fullpath = path.join(_path, configname);
         return fullpath;
+    },
+
+    /**
+     * replaces known variable (i.e. {tf2_dir}) and normalizes the path
+     * @param {string} p path tp expand
+     * @returns {string} replaced and normalized path
+     */
+    ExpandVariables(p){
```

Comment:
Could maybe have a name that better describes whats happening as "expanding" is often used for macros.
Try ResolveVariables or FormatVariables.

Reply:
mostly inspired by the Environment.ExpandEnvironmentVariables in .net ^^
will change to ResolveVariables

Reply:
That's true, but when making this i was think more of string.format from .NET. Shame none keep it standard